polars/polars-lazy/polars-plan/src/logical_plan/format.rs

```diff
@@ -174,7 +174,11 @@ impl LogicalPlan {
             }
             Aggregate {
                 input, keys, aggs, ..
-            } => write!(f, "Aggregate\n\t{:?} BY {:?} FROM {:?}", aggs, keys, input),
+            } => write!(
+                f,
+                "Aggregate\n\t{:?} BY {:?} FROM\n\t{:?}",
```

Comment:
There is an `argument` that should be used. See the other nodes.

The indentation changes when we get deeper in the query.